src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -180,6 +197,10 @@ class BaseOptionsSelector extends Component {
         this.props.onChangeText(value);
     }
 
+    handleReferralModal() {
```

Comment:
Hey there! I'm refactoring this whole component and curious about its purpose. It looks like we never call it.

Reply:
Yes, probably forgot to remove old piece of code